packages/ui-components/src/components/PortalProvider/index.js

```diff
@@ -1,6 +1 @@
-/*
```

Comment:
please return the licenses I guess it is needed

Reply:
Done.